client/devdocs/design/index.jsx

```diff
@@ -125,3 +137,33 @@ export default React.createClass( {
 		);
 	}
 } );
+
+DesignAssets.propTypes = {
```

Comment:
If there is a feature flag, yes, it makes sense, though it makes me question whether we need one. What do you think?